health/health.d/wmi.conf

```diff
@@ -120,14 +120,52 @@ component: Network
        to: sysadmin
 
 
+## Service
+
+ template: wmi_service_status
+       on: wmi.service_status
+    class: Errors
+     type: Windows
+component: Services
+       os: *
+    hosts: *
+ families: *
+   lookup: max -60s unaligned of error,stressed,nonrecover
```

Comment:
All available statuses:

- ok
- error
- unknown
- degraded
- pred_fail
- starting
- stopping
- service
- stressed
- nonrecover
- no_contact
- lost_comm

I was unable to find (despite having a premium subscription to Google Search!) a description/explanation for all of them.

I bet you are too. I doubt we need to add an alarm if we don't understand what those statuses are.

Can you explain what `stressed` and  `nonrecover` are?

Reply:
This is from my personal experience in the past when I worked with Microsoft.

A `stressed` service was a service overload with some event or bad code, in normal situation I saw more the second event.

Now about `nonrecover` this normally happened when developers forgot to recreate index (thanks ancient Borland DBs) and service could not start again. This has relationship with some error in code that does not allow service to restart.